demo/index.html

```diff
@@ -24,51 +24,51 @@
     </div>
 
-    <div class="tiler-viewport" id="slide-horizontal">
+    <div class="tiler-viewport" id="slide-horizontal" data-tiler-animation="slide-horizontal">
       <div class="buttons">
         <h4>Slide</h4>
         <button data-tiler-link="tile-1"></button>
         <button data-tiler-link="tile-2"></button>
         <button data-tiler-link="tile-3"></button>
         <button data-tiler-link="tile-4"></button>
       </div>
-      <div class="tiler-tile" id="tile-1" data-tiler-title="Tile 1" data-tiler-animation="slide-horizontal">
+      <div class="tiler-tile" id="tile-1" data-tiler-title="Tile 1">
         <span id="box-1"></span><span id="box-2"></span><span id="box-3"></span>
       </div>
-      <div class="tiler-tile" id="tile-2" data-tiler-title="Tile 2" data-tiler-animation="slide-vertical"></div>
-      <div class="tiler-tile" id="tile-3" data-tiler-title="Tile 3" data-tiler-animation="slide-horizontal"></div>
-      <div class="tiler-tile" id="tile-4" data-tiler-title="Tile 4" data-tiler-animation="slide-vertical"></div>
+      <div class="tiler-tile" id="tile-2" data-tiler-title="Tile 2"></div>
+      <div class="tiler-tile" id="tile-3" data-tiler-title="Tile 3"></div>
+      <div class="tiler-tile" id="tile-4" data-tiler-title="Tile 4"></div>
     </div>
 
-    <div class="tiler-viewport" id="fade">
+    <div class="tiler-viewport" id="fade" data-tiler-animation="fade">
       <div class="buttons">
         <h4>Fade</h4>
         <button data-tiler-link="tile-1"></button>
         <button data-tiler-link="tile-2"></button>
         <button data-tiler-link="tile-3"></button>
         <button data-tiler-link="tile-4"></button>
       </div>
-      <div class="tiler-tile" id="tile-1" data-tiler-title="Tile 1" data-tiler-animation="fade<">
+      <div class="tiler-tile" id="tile-1" data-tiler-title="Tile 1">
         <span id="box-1"></span><span id="box-2"></span><span id="box-3"></span>
       </div>
-      <div class="tiler-tile" id="tile-2" data-tiler-title="Tile 2" data-tiler-animation="fade<"></div>
-      <div class="tiler-tile" id="tile-3" data-tiler-title="Tile 3" data-tiler-animation="fade<"></div>
-      <div class="tiler-tile" id="tile-4" data-tiler-title="Tile 4" data-tiler-animation="fade<"></div>
+      <div class="tiler-tile" id="tile-2" data-tiler-title="Tile 2"></div>
+      <div class="tiler-tile" id="tile-3" data-tiler-title="Tile 3"></div>
+      <div class="tiler-tile" id="tile-4" data-tiler-title="Tile 4"></div>
     </div>
 
-    <div class="tiler-viewport" id="slide-vertical">
+    <div class="tiler-viewport" id="mixed">
       <div class="buttons">
         <h4>Mixed</h4>
         <button data-tiler-link="tile-1"></button>
         <button data-tiler-link="tile-2"></button>
         <button data-tiler-link="tile-3"></button>
         <button data-tiler-link="tile-4"></button>
       </div>
-      <div class="tiler-tile" id="tile-1" data-tiler-title="Tile 1" data-tiler-animation="fade<">
```

Comment:
no more automatic reversing at all... thats the only part that makes it a major change... reverse animations need defined in css instead of automatically done... too problematic and confusing